ansible/roles/jenkins-worker/templates/aix.rc2.j2

```diff
@@ -17,6 +17,7 @@ start )
               export NODE_TEST_DIR=$HOME/tmp; \
               export NODE_COMMON_PIPE="$HOME/test.pipe"; \
               export OSTYPE=AIX72; \
+              export JOBS=4; \
```

Comment:
Do we want to make this something like 
```suggestion
              export JOBS={{ jobs_env }}; \
```
and set `server_jobs` in the inventory or add a variant in
https://github.com/nodejs/build/blob/ae1841834dfd8d941f356b38f187c07e39131642/ansible/roles/jenkins-worker/vars/main.yml#L142-L149

https://github.com/nodejs/build/blob/ae1841834dfd8d941f356b38f187c07e39131642/ansible/roles/jenkins-worker/tasks/main.yml#L7-L25
This would potentially allow us to set different values on machine basis.